code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -212,12 +212,13 @@
 			else
 				msg += "<b>[t_He] [t_has] severe [damage_desc[CLONE]]!</b>\n"
 
-
+	// TODO: no sense in hardcoding it here, this can be moved onto status effects
```

Comment:
Todo when?

Reply:
my next PR

Reply:
Well you can at least put your examine string in status effect `get_examine_text` rather than adding it here

Reply:
oh that's already a thing? fucking damn

Reply:
Yeah I didn't know that `get_examine_text` was a thing, I thought examine strings had to be added in the examine proc manually